app/styles/ui/_app.scss

```diff
@@ -111,3 +111,13 @@
     }
   }
 }
+
+.sr-only {
```

Comment:
Could we use `screen-reader-only` instead? `sr` was a bit cryptic the first time I saw it 😆 Or maybe `sr` means something else 😳

Reply:
:) - I don't care to change it, I was just copying a standard recommended by accessibility team.

Reply:
Cool! I'll use the same name then.